pkg/agent/route/route_linux.go

```diff
@@ -158,6 +170,42 @@ func (c *Client) initIPTables() error {
 	// Use iptables-restore as it flushes the involved chains and creates the desired rules
 	// with a single call, instead of string matching to clean up stale rules.
 	iptablesData := bytes.NewBuffer(nil)
+	// Write head lines anyway so the undesired rules can be deleted when changing encap mode.
+	writeLine(iptablesData, "*raw")
+	writeLine(iptablesData, iptables.MakeChainLine(antreaPreRoutingChain))
+	writeLine(iptablesData, iptables.MakeChainLine(antreaOutputChain))
+	if c.networkConfig.TrafficEncapMode.SupportsEncap() {
+		// For Geneve and VXLAN encapsulation packets, the request and response packets don't belong to a UDP connection
+		// so tracking them doesn't give the normal benefits of conntrack. Besides, kube-proxy may install great number
+		// of iptables rules in nat table. The first encapsulation packets of connections would have to go through all
+		// of the rules which wastes CPU and increases packet latency.
+		udpPort := 0
+		if c.networkConfig.TunnelType == ovsconfig.GeneveTunnel {
+			udpPort = 6081
```

Comment:
Define two constants for the port numbers?

Reply:
sure, will do